### PR TITLE
Feat - Remove Libraries Section

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

Until we have some content for the libraries section:

* Removed the libraries section as it's the same content as in the home page.  
* Added # link to the home page until we have proper libraries content.

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Wiki was built locally

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ module.exports = {
         },
         {
           label: 'Libraries',
-          to: 'categories/libraries',
+          to: '/#libraries',
```
Comment (aleksei-korolev) [inline docusaurus.config.js:95]: Out of curiosity, what does that change do?

Comment (Dr-Electron) [reply docusaurus.config.js:95]: It scrolls to `Libraries` on the landing page

Comment (aleksei-korolev) [reply docusaurus.config.js:95]: right :facepalm:

**Review by aleksei-korolev — APPROVED** [review]

**Review by Dr-Electron — APPROVED** [review]
LGTM. Although I can't remember why we decided against just linking to the landing page. But I think this is fine 😅

**Comment by jlvandenhout** [comment]
> LGTM. Although I can't remember why we decided against just linking to the landing page. But I think this is fine 😅

Because the landing page is just a showcase and not necessarily a complete reference to everything. So I wanted to have this section already as a separate page, so we can improve upon it and have a place to write content to @lucas-tortora . But if we want content first, before creating the section, that's fine by me.

**Pull request merged**